specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2023-05-01-privatepreview/AlertProcessingRules.json

```diff
@@ -0,0 +1,865 @@
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/errorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DeleteAlertProcessingRule": {
+            "$ref": "./examples/AlertProcessingRules_Delete.json"
+          }
+        }
+      },
+      "patch": {
+        "description": "Enable, disable, or update tags for an alert processing rule.",
+        "operationId": "AlertProcessingRules_Update",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "name": "alertProcessingRuleName",
+            "in": "path",
+            "description": "The name that needs to be updated.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "alertProcessingRulePatch",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PatchObject"
+            },
+            "description": "Parameters supplied to the operation."
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Returns the created/updated alert processing rule.",
+            "headers": {
+              "x-ms-request-id": {
+                "type": "string",
+                "description": "Service generated Request ID."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/AlertProcessingRule"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/errorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "PatchAlertProcessingRule": {
+            "$ref": "./examples/AlertProcessingRules_Patch.json"
+          }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "subscriptionId": {
+      "name": "subscriptionId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The ID of the target subscription.",
+      "minLength": 1
+    },
+    "resourceGroupName": {
+      "name": "resourceGroupName",
+      "description": "Resource group name where the resource is created.",
+      "type": "string",
+      "in": "path",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "api-version": {
+      "name": "api-version",
+      "description": "Client API version.",
+      "type": "string",
+      "in": "query",
+      "required": true,
+      "enum": [
+        "2023-05-01-privatepreview"
```

Comment:
Is "2021-08-08" already deprecated? If not, a user can't use that API version anymore, is it?

Reply:
I updated the file to reference the common type.